### ISSUE TITLE
the classification loss is halt when the dataset of your paper is trained in other model

### ISSUE THREAD

**Issue by JoyLuo** [ShuangLI59/person_search#24]
Dear Tong xiao:
       thank for you dataset.
       and when i use the dataset to train other net, the loss of classification can not be down normally. 
       because you are familiar with the dataset, and the paper named《End-to-End Deep Learning for Person Search》has mention this "When combined together, these two issues make it extremely difficult for the net to receive proper gradients. In practice, we found that the training loss would not decrease if we directly finetune the network from the ImageNet pre-trained VGG16 model."
       could you give me some idea if you have time, thanks very much.

**Comment by Cysu** [comment]
Yes. We did find that training in a Faster-RCNN framework with softmax loss does not work, which inspired us to design new loss functions.

However, I could not figure out your problem given such limited information. Maybe you could refer to our new version of paper [Joint Detection and Identification Feature Learning for Person Search](https://arxiv.org/pdf/1604.01850.pdf) and see if it could solve your problem.
